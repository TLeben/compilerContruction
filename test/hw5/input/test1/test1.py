
a = lambda : input() + input()

print a()