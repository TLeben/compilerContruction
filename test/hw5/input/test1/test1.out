[4, 6, 9]
True
